### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by badger-99**
This PR will introduce the following features to the `dev` branch:

- A form to create a new post.
- A form to create a new comment
- A like button

### CODE REVIEW

File: config/routes.rb
Comment (Owusu-Desmond) [inline config/routes.rb:1]: - [ ] Kindly make sure that you have this route `GET /posts/:post_id/comments/new(.:format) comments#new` that will be used for displaying the comment form. :pray:

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
```
Comment (Owusu-Desmond) [inline app/controllers/comments_controller.rb:1]: - [ ] Please ensure that you have a new action in your comment controller, which is to display a form creating a comment that should be created in `/views/comments/new.html.erb` for creating a new instance of a model. ( In other words, when a user wants to create a new comment in the database.)

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @badger-99 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @badger-99, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Requested change implemented 🔥 


## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**